code/__DEFINES/maths.dm

```diff
@@ -228,7 +228,7 @@
 #define EXP_DISTRIBUTION(desired_mean) ( -(1/(1/desired_mean)) * log(rand(1, 1000) * 0.001) )
 
 #define LORENTZ_DISTRIBUTION(x, s) ( s*tan(TODEGREES(PI*(rand()-0.5))) + x )
-#define LORENTZ_CUMULATIVE_DISTRIBUTION(x, y, s) ( (1/PI)*TORADIANS(arctan((x-y)/s)) + 1/2 )
+#define LORENTZ_CUMULATIVE_DISTRIBUTION(x, y, s) ( (1/PI)*arctan((x-y)/s) + 1/2 )
```

Comment:
This is intentional. The macro was previously used to convert the probability density to comulative distribution, which justified the added TORADIANS macro in the formula. Now that it's no longer does so, this has got to go.